itou/www/approvals_views/views.py

```diff
@@ -248,9 +248,11 @@ def get_context_data(self, **kwargs):
             if any(suspension.is_in_progress for suspension in long_suspensions):
                 approval_can_be_deleted = True
             elif long_suspensions:
-                last_hiring = approval.jobapplication_set.accepted().order_by("hiring_start_at").last()
-                if last_hiring is None or any(
-                    suspension.end_at > last_hiring.hiring_start_at for suspension in long_suspensions
+                last_hiring_start_at = approval.jobapplication_set.accepted().aggregate(Max("hiring_start_at"))[
```

Comment:
Pas plus simple de faire ça ?
```suggestion
                last_hiring = approval.jobapplication_set.accepted().latest("hiring_start_at")
```

Reply:
Il faudrait aussi rajouter un `.exclude(hiring_start_at=None)`.
Cf la note de https://docs.djangoproject.com/en/5.1/ref/models/querysets/#django.db.models.query.QuerySet.latest `earliest() and latest() may return instances with null dates.`
Et je me dis que c'est plus joli de ne demander que la date plutôt que de récupérer toutes les infos de la candidature.